src/patterns/menu/Menu.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useRef, useState } from 'react';
+import type { Property } from 'csstype';
+
+import MenuItem from '../../components/menu-item';
+import Icon from '../../foundations/icon';
+import Shape from '../../foundations/shape';
+import Popover from '../../utilities/popover';
+import Grid from '../../utilities/grid';
+
+import MenuRoot, { MenuLabelRoot } from './Menu.root';
+import type { PSubmenu } from './Menu.props';
+import type PMenu from './Menu.props';
+import { MENU_TRANSITIONS_MAP } from './Menu.constants';
+
+const Divider = (): JSX.Element => (
+    <Shape height="1px" width={'auto'} backgroundColor={'#e0e0e0'} />
+);
+
+const mobileMenuStyles = {
```

Comment:
when you type the const as `React.CSSProperties` you do not have to cast `Property.Position` on the `position`